maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/ServerHelper.java

```diff
@@ -0,0 +1,44 @@
+package org.openmrs.maven.plugins.utility;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * Created by Gregor Karetka on 27.2.2017.
+ */
+public class ServerHelper {
+    private Wizard wizard;
+    private static final int MAX_USHORT_VALUE = 65535;
+
+    public ServerHelper(Wizard wizard) {
+        this.wizard = wizard;
+    }
+
+    public boolean isPortInUse(int port) {
+        ServerSocket serverSocket = null;
+        try {
+            serverSocket = new ServerSocket(port);
+            this.closeTestingServerSocket(serverSocket);
+        } catch (IOException e) {
+            return true;
+        }
+        return false;
+    }
+
+    private void closeTestingServerSocket(ServerSocket serverSocket) throws IOException {
+        serverSocket.close();
+
+        wizard.showMessage("Closing test socket");
+        while (!serverSocket.isClosed()) {}
```

Comment:
Could we add at least Thread.sleep(500) so that it doesn't unnecessarily strain the system?